### PR TITLE
Added Week 5 Content

### PR DESCRIPTION
**Pull request by quarbby**

### CODE REVIEW

**Comment by damithc** [comment]
Can limit lines to about 110 chars where possible? Reduce excessive indentation too.

**Comment by quarbby** [comment]
Limited lines prof!

File: contents/week5.html [hunk 1]
```diff
@@ -37,81 +43,213 @@ <h3 id="things-to-do-content-week5" class="buttoned"><span>Things to do</span></
...
+            <h3>[Individual, 3 pts] Activity 3 </h3>
+            <div>
+				Propose an architecture for your product. 
+				i.e. Divide the product into a small number of components. 
+				Discover some of its component APIs. 
+				Where applicable, draw some sequence diagrams (at least one) to support the discovery of APIs. 
+				Keep the hard/soft copies of your diagrams to bring to the tutorial (rough hand-drawn sketches are OK too).
+				If you are not very sure about how to draw sequence diagrams, you can do the following optional 
+				team exercise to increase your confidence before attempting the above task.			
+			</div>
```
Comment (damithc) [inline contents/week5.html]: indentation is off for the text as well as the closing tag.

File: contents/week5.html [hunk 2]
```diff
@@ -37,81 +43,213 @@ <h3 id="things-to-do-content-week5" class="buttoned"><span>Things to do</span></
...
+        </div>		
+		<div class="homework homework-week5">
+			<h3> [Team, Optional] Activity 4</h3>
+			<div>
+			Propose an architecture for the text-based Minesweeper game used as an example in lecture 4. 
+			Discover the API of its components using sequence diagrams. 
+			</div>
+		</div>
```
Comment (damithc) [inline contents/week5.html]: too much indentation. use 4 spaces to indent.

**Comment by damithc** [comment]
Have another look through. There are indentation misalignments. Use 4 spaces to indent. Too much indentation is especially a problem for html files due to deep nesting.

**Comment by quarbby** [comment]
I hope it's correct now :)

**Comment by damithc** [comment]
Remove thumbs.db

**Comment by quarbby** [comment]
Removed Thumbs.db

**Comment by damithc** [comment]
Give good headings for these ('Activity 1' etc was just a place holder)
image
Images should be moved to the previous section.
image

**Comment by damithc** [comment]
Part 1 does not expand when I click the button. As it works in week 2, I'm guessing it's a mistake in your code rather than the script, but I could be wrong.
image

**Comment by damithc** [comment]
Is this ready for review?

**Comment by quarbby** [comment]
Yes, you may review it.

**Pull request merged**